src/eslint-plugin-sx/src/getRules.js

```diff
@@ -11,7 +11,7 @@ module.exports = function getRules() /*: {| +[ruleName: string]: string |} */ {
     withFileTypes: true,
   })) {
     // $FlowIssue[prop-missing]: https://github.com/facebook/flow/pull/8532
-    if (dirent.isFile()) {
+    if (dirent.isFile() && /\.js$/.test(dirent.name)) {
```

Comment:
Would it be possible to test this somehow, in case someone tries to change it and possibly breaks it again?

Reply:
Added tests, please have a look again. :)